slack-sup/models/sup.rb

```diff
@@ -87,6 +87,13 @@ def calendar_href(dt = nil)
 
   private
 
+  def select_best_captain
+    users.sort_by do |u|
+      last_captain_at = u.last_captain_at
+      [last_captain_at ? 1 : 0, last_captain_at]
```

Comment:
had to do this so users who haven't been captain with `last_captain_at` being `nil` end up in the beginning of the list, any better way to do this?

Reply:
You can probably just `return u unless u.last_captain_at` inside the loop?

Reply:
don't think i can do that, i get 
```ruby
ArgumentError:
       comparison of NilClass with User failed
```
there must be a more obvious way, i've seen people do something like
```ruby
( users.select(&:last_captain_at?).sort_by(&:last_captain_at) + users.reject(&:last_captain_at?) ).last
```
but this calls `last_captain_at` for a user twice